FirebaseAdmin/FirebaseAdmin.Tests/FirebaseAdmin.Tests.csproj

```diff
@@ -28,6 +28,7 @@
     <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.556">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <AdditionalFiles Include="../../stylecop.json" />
```

Comment:
Do you know what triggered this change?

Reply:
Added this to apply stylecop.json custom settings to test files.